### PR TITLE
feat: install and update design system with tailwind

### PR DESCRIPTION
**Pull request by jasonzhwang**
Tailwind is installed, and design system has been refactored.

1. preset tailwind style is imported into _app with old sass styles. Please remove old sass styles related to your own future refactor ticket. Sass files should be all gone, once all refactor ticket is finished.
image

2.heading style is preset in tailwind, other text styles can refer from design system page.
image

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 
 import type { AppProps } from 'next/app'
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return(
+  <>
+    <Head>
+      <title>Taotify</title>
+      <meta name="description" content="Generated by create next app" />
```
Comment (tatoMa) [inline src/pages/_app.tsx:15]: Since you have added this description, please make it meaningful.

**Review by tatoMa — COMMENTED** [review]
Nice work overall.

File: src/pages/_app.tsx (hunk 1 above)
Comment (jasonzhwang) [reply src/pages/_app.tsx]: update description with "Discover, Play, and Enjoy Your Favorite Music with Taotify - Your Ultimate Music Companion"

**Review by HjsCS — APPROVED** [review]
LGTM

**Pull request merged**